src/typescript_proto_library.bzl

```diff
@@ -97,7 +100,11 @@ def _get_outputs(target, ctx):
     js_outputs_es6 = []
     dts_outputs = []
     for src in target[ProtoInfo].direct_sources:
-        file_name = src.basename[:-len(src.extension) - 1]
+        if ctx.label.workspace_root == "":
```

Comment:
Can you add a comment describing when workspace_root is empty?

Reply:
Done